evap/staff/templates/staff_single_result_form.html

```diff
@@ -11,12 +11,16 @@
 
         <div class="form-group well">
             <div class="col-sm-offset-2 col-sm-6">
-                <input type="submit" value="{% trans "Save course" %}" class="btn btn-primary" onclick="preventAccidentalClosing = false;"/>
+                {% if form.instance.can_staff_edit %}
+                    <input type="submit" value="{% trans "Save course" %}" class="btn btn-primary" onclick="preventAccidentalClosing = false;"/>
+                {% endif %}
```

Comment:
oh, i see. nevermind, for staff_course_form we don't need it because the save button is always there.